### ISSUE TITLE
Implement a as( XCMSnExp) for xcmsSet

### ISSUE THREAD

**Issue by stanstrup** [sneumann/xcms#601]
I know the reverse exist but any chance to convert xcmsSet to XCMSnExp?

I have an mzMine peaklist I need to process. I have an old `mzmine2xcmsSet` function but need XCMSnExp downstream :(

**Comment by sneumann** [comment]
I don't dare to suggest, but wouldn't mzmine2XCMSnExp be the cleaner solution, and more sustainable in the long run (less moving parts inside) ? Yours, Steffen

**Comment by stanstrup** [comment]
Yes I was starting to think the same actually... If there is not really another use case for `as(.., "XCMSnExp")`.
